upstream-tools/replacements.json

```diff
@@ -3455,6 +3455,18 @@
     {
       "old": "  name                     = \"terraform-example-collaboration\"",
       "new": "  name                     = \"pulumi-example-collaboration\""
+    },
+    {
+      "old": "% terraform import aws_cleanrooms_collaboration.collaboration 1234abcd-12ab-34cd-56ef-1234567890ab",
+      "new": "% TODO import aws_cleanrooms_collaboration.collaboration 1234abcd-12ab-34cd-56ef-1234567890ab"
```

Comment:
I believe that we are supposed to replace these `"new": TODO`s manually, then re-run `make upstream`.